commons-api/src/main/java/com/connexta/transformation/commons/api/status/TransformationStatus.java

```diff
@@ -24,7 +24,6 @@
  * the transformation the status is associated with.
  */
 public interface TransformationStatus {
-
```

Comment:
Do we want this line removed?

Reply:
my formatter removes that one whenever I end up touching anything close to that location 😄